app/components/counter/Counter.js

```diff
@@ -0,0 +1,190 @@
+    const margin = { margin: '15px' }
+    const svg = { height: '36px', width: '36px' }
+    const font = { fontSize: '7em' }
+    const showing = { display: this.state.hidden ? 'none' : 'block' }
+    const emoji = { height: '16px', width: '16px' }
+
+    return (
+      <div>
+        <Paper style={ {width: 300, height: 400} }>
+
+          <div className="row" style={center}>
+            {/* Numeric Countdown */}
+            <p style={ font }> { mins }:{ secs < 10 ? `0${secs}` : secs } </p>
+          </div>
+
+          <div className="row" style={center}>
+            {/* Greenish circular button */}
+            <FloatingActionButton onClick={ () => this.onStartClick() } style={ margin }>
+              <PlayCircleOutline style={ svg } />
+            </FloatingActionButton>
+            {/* Reddish circular button */}
+            <FloatingActionButton onClick={ () => this.onStopClick() } secondary={ true } style={ margin }>
+              <Close style={ svg } />
+            </FloatingActionButton>
+          </div>
+
+          {/* Emoji pain menu and comment form hidden until time stopped */}
+          <div className="row" style={ showing }>
+            <div style={ center }>
+              <IconButton
+                tooltip="No Pain"
+                tooltipPosition="bottom-center"
+                onClick={ () => this.onPainClick(1) }
+              >
+                <img src={require("../../../src/images/emojis/1pain.svg")} style={ emoji } />
+              </IconButton>
+              <IconButton
+                tooltip="Minor Pain"
+                tooltipPosition="bottom-center"
+                onClick={ () => this.onPainClick(2) }
+              >
+                <img src={require("../../../src/images/emojis/2pain.svg")} style={ emoji } />
+              </IconButton>
+              <IconButton
+                tooltip="Some Pain"
+                tooltipPosition="bottom-center"
+                onClick={ () => this.onPainClick(3) }
+              >
+                <img src={require("../../../src/images/emojis/3pain.svg")} style={ emoji } />
+              </IconButton>
+              <IconButton
+                tooltip="Major Pain"
+                tooltipPosition="bottom-center"
+                onClick={ () => this.onPainClick(4) }
+              >
+                <img src={require("../../../src/images/emojis/4pain.svg")} style={ emoji } />
+              </IconButton>
+              <IconButton
+                tooltip="Serious Pain"
+                tooltipPosition="bottom-center"
+                onClick={ () => this.onPainClick(5) }
+              >
+                <img src={require("../../../src/images/emojis/5pain.svg")} style={ emoji } />
+              </IconButton>
+            </div>
+            <div style={ center }>
+              <TextField
+                hintText="Any notes from the workout?"
+                floatingLabelText="Comments"
+                value={ this.state.comments }
```

Comment:
I don't think this value is needed, the field will work without it passed back down